src/main/java/longah/util/MemberList.java

```diff
@@ -143,27 +143,27 @@ public ArrayList<Subtransaction> solveTransactions() {
 
             // Check the current pair for which balance is greater or if equal
             if (positiveBalance > negativeBalance) {
-                Subtransaction subtransaction = 
-                        new Subtransaction(negativeMember, positiveMember,
-                        negativeBalance);
+                Subtransaction subtransaction =
+                        new Subtransaction(positiveMember, negativeMember,
+                                negativeBalance);
```

Comment:
i not sure about coding standard here, shouldn't it follow the already indented indentation